workflow-templates/php-library-security.yml

```diff
@@ -0,0 +1,17 @@
+name: Security analysis
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  security:
+    uses: dvsa/.github/.github/workflows/php-security.yml@v3.2.0
```

Comment:
Should this be using `php-library-security`? Same question for static and test workflow templates

Reply:
yes